modules/points.py

```diff
@@ -39,54 +27,57 @@ def install(self, application: Application) -> None:
         application.add_handler(CommandHandler("donate", self.__donate))
```

Comment:
install function is reccurent in other modules. We should have them in a parent class and make those modules classes as children

Reply:
You are right about this.

Right now I'm experimenting with the [ConversationHandler](https://docs.python-telegram-bot.org/en/stable/telegram.ext.conversationhandler.html) functionality, and part of this will imply changes to the install function. Once that part is done I will add a base class / interface to handle modules.